pallets/confidential/src/lib.rs

```diff
@@ -70,6 +75,10 @@ decl_storage! {
         pub RangeProofs get(fn range_proof): double_map hasher(twox_64_concat) IdentityId, hasher(blake2_128_concat) ProverTickerKey => Option<TickerRangeProof>;
 
         pub RangeProofVerifications get(fn range_proof_verification): double_map hasher(blake2_128_concat) (IdentityId, Ticker), hasher(twox_64_concat) IdentityId => bool;
+
+        pub MercatAccounts get(fn mercat_accounts): map hasher(blake2_128_concat) IdentityId => Vec<PubAccount>;
```

Comment:
This data structure is supposed to hold the list of all the meract accounts that an identity holds. Is this the right place for storing it? Or should be be moved to the `asset` pallet?

Reply:
I am more inclined to use the different pallets i.e `confidential-asset`. We can discuss more about this if you let me know the expected API for the confidential asset.

Reply:
We'll be spending some time in the next couple of days figuring out the API and the storage that we need on the Polymesh. We can revisit this discussion once that is done.